llvm/lib/SYCLLowerIR/ComputeModuleRuntimeInfo.cpp

```diff
@@ -155,6 +155,35 @@ std::optional<T> getKernelSingleEltMetadata(const Function &Func,
   return std::nullopt;
 }
 
+PropSetRegTy computeBF16DeviceLibProperties(const Module &M,
+                                            std::string &BF16DeviceLibName) {
+  PropSetRegTy PropSet;
+
+  {
+    for (const auto &F : M.functions()) {
+      if (!F.getName().starts_with("__devicelib_") || F.isDeclaration())
```

Comment:
We should swap the condition to reduce amount of string comparisons

Reply:
Done.
Thanks very much.